pkg/collector/corechecks/servicediscovery/usm/service_test.go

```diff
@@ -214,10 +214,21 @@ func TestExtractServiceMetadata(t *testing.T) {
 			name: "node js with a broken package.json",
 			cmdline: []string{
 				"/usr/bin/node",
-				"./testdata/inner/index.js",
+				"./testdata/inner/app.js",
 			},
 			lang:                  language.Node,
-			expectedGeneratedName: "node",
+			expectedGeneratedName: "app.js",
```

Comment:
should it be `app.js` and not `app`?
for python we're stripping the suffix

Reply:
Changed it to strip the suffix for consistency with Python.